docker/mercure/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM dunglas/mercure
```

Comment:
@mwarzybok-sumoheavy let's specify `dunglas/mercure:v0.15` since we don't know if v0.16 will be backwards compatible.